plugins/net_plugin/net_plugin.cpp

```diff
@@ -412,10 +415,12 @@ namespace eosio {
     *  the need for compatibility hooks
     */
    constexpr uint16_t proto_base = 0;
-   constexpr uint16_t proto_explicit_sync = 1; // version at time of eosio 1.0
-   constexpr uint16_t block_id_notify = 2;     // reserved. feature was removed. next net_version should be 3
+   constexpr uint16_t proto_explicit_sync = 1;       // version at time of eosio 1.0
+   constexpr uint16_t proto_block_id_notify = 2;     // reserved. feature was removed. next net_version should be 3
+   constexpr uint16_t proto_pruned_types = 3;        // supports new signed_block & packed_transaction types
+   constexpr uint16_t proto_generic_messages = 4;    // placeholder for generic messages
```

Comment:
generic_messages not going in, so remove this

Reply:
removed

Reply:
So you are still keeping the placeholder for it?

Reply:
No, I removed it.